eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,8 +1,33 @@
+resources:
+  repositories:
+    - repository: azure-sdk-tools
+      type: github
+      name: azure/azure-sdk-tools
+      endpoint: azure
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: refs/tags/azure-sdk-build-tools_20200514.1
+
 parameters:
-  Artifacts: []
-  ServiceDirectory: not-specified
-  TargetDocRepoOwner: 'Azure'
-  TargetDocRepoName: 'azure-docs-sdk-dotnet'
+- name: Artifacts
```

Comment:
Do these all have to be explicitly defined now? What caused you to switch to long form?

Reply:
Yes, this is required for extending the template.